lib/Cleantalk/ApbctWP/Activator.php

```diff
@@ -3,6 +3,7 @@
 namespace Cleantalk\ApbctWP;
 
 use Cleantalk\ApbctWP\AdjustToEnvironmentModule\AdjustToEnvironmentHandler;
+use Cleantalk\ApbctWP\ServerChecker\ServerChecker;
```

Comment:
Naming: Can you rename the class to Server**Requirements**Checker?

This name is further specialized to reflect the class purposes.